index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="shortcut icon" href="images/Apple-Logo.png" type="image/x-icon">
+  <link rel="stylesheet" href="normalize.css">
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Apple Clone</title>
+</head>
+<body>
+  <main class="container"> <!--Container Start-->
+    <div class="nav-flex"> <!--Navbar items Start-->
+      <div class="nav-item nav-item-1"><img class="apple-logo" src="images/apple_logo.png" alt=""></div>
+      <div class="nav-item nav-style nav-item-2">Store</div>
+      <div class="nav-item nav-style nav-item-3">Mac</div>
+      <div class="nav-item nav-style nav-item-4">iPod</div>
+      <div class="nav-item nav-style nav-item-5">iPad</div>
+      <div class="nav-item nav-style nav-item-6">iTunes</div>
+      <div class="nav-item nav-style nav-item-7">Support</div>
+      <div class="nav-item nav-item-8">
+        <form action="#">
+          <input type="text">
+        </form>
+      </div>
+    </div><!--Navbar Items End-->
+    <div class="main-text-header"> <!--Container which contains my header-->
+    </div>
+    <div class="secondary-text"><!--Container for my secondayr text-->
+    </div>
+    <div class="your-verse"> <!--Container for the last text inside main-->
+    </div>
+  </main><!--Container end-->
+  <div class="flex-container"><!--Flex Container for secundary options start-->
```

Comment:
[OPTIONAL]
You did a nice job using HTML5 semantic tag. You can also make this one be a `section` tag instead

Reply:
Okay. Thanks for your review! XD